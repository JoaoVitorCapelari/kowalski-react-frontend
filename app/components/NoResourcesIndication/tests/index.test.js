// import React from 'react';
// import { shallow } from 'enzyme';

// import NoResourcesIndication from '../index';

describe('<NoResourcesIndication />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
