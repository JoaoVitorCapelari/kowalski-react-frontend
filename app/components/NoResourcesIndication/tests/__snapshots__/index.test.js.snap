// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResourcesIndication /> renders 1`] = `"<div class=\\"NoResourcesIndication__Container-ljeacB bckeoK\\"><span>There are no users registered :(</span></div>"`;
